### PR TITLE
Renamed JoplinCGE datatype to CGE

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: datatypes/dataservice_pull_down_menu.md [hunk 1]
```diff
@@ -123,20 +123,20 @@ are not included in Data service.
...
-- incore:JoplinCGEjobcr
-- incore:JoplinCGElandcap
-- incore:JoplinCGEmisc
-- incore:JoplinCGEmisch
-- incore:JoplinCGEoutcr
-- incore:JoplinCGEsam
-- incore:JoplinCGEshocks
-- incore:JoplinCGEtauff
-- incore:JoplinCGEtpc 
+- incore:CGE
```
Comment (navarroc) [inline datatypes/dataservice_pull_down_menu.md:126]: Is there an incore:CGE type?

Comment (ywkim312) [reply datatypes/dataservice_pull_down_menu.md:126]: no, it was always incore:CGE+something. so eventually, all.md or dataservice_pull_down_menu file, there is only one incoer:CGE

Comment (navarroc) [reply datatypes/dataservice_pull_down_menu.md:126]: Should that be removed then? This seems to imply there is an incore:CGE type.

Comment (ywkim312) [reply datatypes/dataservice_pull_down_menu.md]: removed from both all and dataservice_pull_down_menu

**Review by longshuicy — APPROVED** [review]

**Pull request merged**